From 2bd9d5de884685ea51181438cc0674a1f955b1cf Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Mon, 19 Nov 2012 15:23:56 +0200
Subject: [PORT FROM MAIN] Telephony: Increase the Send SMS retry delay timer

BZ: 22238

When the device is registered to the network,framework will
start the SMS retry. Number of retry is 3 and the retry interval
is 2seconds. Since the retry interval is very less, some of the
SMS are never sent out. Failure may be due to modem busy with SIM
files caching or service center busy etc.

This patch increases the SEND_RETRY_DELAY to 10seconds.

Change-Id: Ia20e0ca07eff33dfbcbb436a4c6856874f4b106f
Orig-Change-Id: I70160ca6a37cfdfb1c94ac39f5fd31f24ab249a7
Category: device enablement
Domain: Telephony-Telephony-FWK
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Jeevaka Badrappan <jeevaka.badrappan@intel.com>
---
 .../android/internal/telephony/SMSDispatcher.java  |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/java/com/android/internal/telephony/SMSDispatcher.java b/src/java/com/android/internal/telephony/SMSDispatcher.java
index 9cd5b70..0258ff3 100644
--- a/src/java/com/android/internal/telephony/SMSDispatcher.java
+++ b/src/java/com/android/internal/telephony/SMSDispatcher.java
@@ -133,7 +133,7 @@ public abstract class SMSDispatcher extends Handler {
     /** Maximum number of times to retry sending a failed SMS. */
     private static final int MAX_SEND_RETRIES = 3;
     /** Delay before next send attempt on a failed SMS, in milliseconds. */
-    private static final int SEND_RETRY_DELAY = 2000;
+    private static final int SEND_RETRY_DELAY = 10000;
     /** single part SMS */
     private static final int SINGLE_PART_SMS = 1;
     /** Message sending queue limit */
-- 
1.7.4.1

